README.md

```diff
@@ -1,5 +1,16 @@
 # The ix modeling platform (ixmp)
 
+## TLDR
+
+```bash
+# setup python2/3 virtual environment
```

Comment:
Perhaps add a note that this is optional? It is also not exactly a tl;dr because users will have to install all the other non-trivial dependencies. 

@danielhuppmann should comment on how/where he thinks this should be placed.